src/protocol/payload/addr.rs

```diff
@@ -124,7 +124,7 @@ impl Codec for NetworkAddr {
     }
 
     fn decode<B: Buf>(bytes: &mut B) -> io::Result<Self> {
-        let timestamp = read_timestamp(bytes)?;
+        let timestamp = read_short_timestamp(bytes)?;
```

Comment:
This protocol change occured without any change to the message command string in the header?

Reply:
I think `addr` message hasn't worked in tests before (possibly due to a long reply time?), so it's the same message as before - it might be the only one with a timestamp this short.